MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -19,14 +19,19 @@ final class FeedViewModel {
 	}
 
 	var onLoadingStateChange: Observer<Bool>?
+	var onErrorMessageChange: Observer<String?>?
```

Comment:
`onErrorMessageChange` could be named `onErrorStateChange` following the `onLoadingStateChange` property.

```suggestion
	var onErrorStateChange: Observer<String?>?
```